### PR TITLE
[4.0] Add missing CSS variables to cassiopeia

### PR DESCRIPTION
**Pull request by laoneo**
Pull Request for Issue #34546 and #34545.

### Summary of Changes
Adds the missing variables to cassiopeia to style the media manager properly on the front.

**Please comment and test this one not before #34621 is merged!!!**

### Testing Instructions
- Set "Frontend images" to show in the articles options.
- Edit an article on the front.
- Click on "Images".
- Click on "Select"

### Actual result BEFORE applying this Pull Request
Media manager has limited styling.

### Expected result AFTER applying this Pull Request
Media manager has proper styling, similar to back end.

### Documentation Changes Required
Basically this pr adds the missing variables for front end templates. Can be used for proper documentation.

### CODE REVIEW

**Comment by brianteeman** [comment]
Nice work.

As these are not user editable in cassiopeia would it not be better to have them in a stylesheet instead of as an inline style. If this stylesheet was located in the /templates/system folder instead of the /templates/cassiopeia folder then it would be even better as it would be more _obvious_ that a template developer needs to include it in their own template in order for media manager to work correctly in the front end

**Comment by dgrammatiko** [comment]
It's fine as inline styles, but this needs to be documented that AFFECTS every front end template.
Also, a personal preference would be to change the prefix from `template` to `jdt` (joomla design token) or something similar. The template prefix is not really representative that these are Global design tokens. But maybe that's just me.

**Comment by brianteeman** [comment]
> It's fine as inline styles, but this needs to be documented that AFFECTS every front end template.

Doing it in a stylesheet will mean that it can be updated in a new release without having to ask every template developer and site owner to update their own templates.

So wouldn't this be more obvious (and useful) as every template will need it

_Pseudo code untested_
```
// Enable assets
$wa->usePreset('template.cassiopeia.' . ($this->direction === 'rtl' ? 'rtl' : 'ltr'))
	->useStyle('template.active.language')
	->useStyle('template.user')
	->useScript('template.user')
	// global design classes
	->useStyle('template.global');
```

**Comment by laoneo** [comment]
So this one can be tested now as #34621 got merged.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on d2968b56ed90f229c463e4a191d871f182347c0c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34624">issues.joomla.org/tracker/joomla-cms/34624</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on d2968b56ed90f229c463e4a191d871f182347c0c<br /><br />thank you<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34624">issues.joomla.org/tracker/joomla-cms/34624</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34624">issues.joomla.org/tracker/joomla-cms/34624</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!
